src/assets/scss/components/_buttons.scss

```diff
@@ -0,0 +1,24 @@
+@import 'variables';
+
+.button {
+  border-radius: 2px;
+  text-decoration: none;
+  box-shadow: none;
+  font-size: $sub-baseline * 1.5;
+  padding: $sub-baseline $baseline;
+  text-decoration: none;
```

Comment:
oops text decoration is twice in this ruleset

Reply:
Good eye!